packages/components/src/components/Note/stories/Default.stories.tsx

```diff
@@ -0,0 +1,55 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import Note from "../Note";
+import React from "react";
+import { Heading } from "@/components/Heading";
+import { Content } from "@/components/Content";
+import { Icon } from "@/components/Icon";
+import { faEnvelope } from "@fortawesome/free-regular-svg-icons";
+
+const meta: Meta<typeof Note> = {
+  title: "Note",
+  component: Note,
+  argTypes: {
+    variant: {
+      control: "inline-radio",
+    },
+  },
+  args: { variant: "info" },
```

Comment:
Fehlen hier noch Varianten?

Reply:
ich hatte die Varianten Stories in eine eigene Datei gepackt, so wie beim Button